oc_dms_mirror/rest_api/app/routes.py

```
@@ -1,68 +1,92 @@
+            content_type='application/json',
+            response=data)
+
+    def register_component_version_artifact(self):
+        """
+        Endpoint performing component/version sync with DMS on demand.
+        """
+        self.logger.info(f"POST {request.url_rule.rule} from [{request.remote_addr}] with payload: {request.get_json()}")
+        try:
+            self.dms_mirror.process_component_webhook(request.get_json())
+        except Exception as _e:
+            self.logger.exception(_e)
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)